pallets/identity/src/lib.rs

```diff
@@ -729,3 +718,86 @@ fn revoke_claim_class(claim_type: ClaimType) -> frame_support::weights::Dispatch
         _ => Normal,
     }
 }
+
+pub mod migration {
+    use super::*;
+
+    mod v1 {
+        use super::*;
+        use polymesh_primitives::secondary_key::v1;
+        use scale_info::TypeInfo;
+
+        /// Old v1 Identity information.
+        #[derive(Encode, Decode, TypeInfo)]
+        #[derive(Clone, Default, PartialEq)]
+        pub struct IdentityRecord<AccountId> {
+            pub primary_key: AccountId,
+            pub secondary_keys: Vec<v1::SecondaryKey<AccountId>>,
+        }
+
+        decl_storage! {
+            trait Store for Module<T: Config> as Identity {
+                pub DidRecords get(fn did_records): map hasher(identity) IdentityId => IdentityRecord<T::AccountId>;
+                pub KeyToIdentityIds get(fn key_to_identity_dids):
+                    map hasher(twox_64_concat) T::AccountId => IdentityId;
+            }
+        }
+
+        decl_module! {
+            pub struct Module<T: Config> for enum Call where origin: T::Origin { }
+        }
+    }
+
+    pub fn migrate_v1_key<T: Config>(key: T::AccountId, record: KeyRecord<T::AccountId>) {
```

Comment:
```suggestion
    fn migrate_v1_key<T: Config>(key: T::AccountId, record: KeyRecord<T::AccountId>) {
```

Reply:
No.  This is used by the MultiSig migration.